docker-compose.yml

```diff
@@ -1,11 +1,10 @@
-version: '2'
 services:
   tests:
-    image: buildkite/plugin-tester
+    image: buildkite/plugin-tester:v4.2.0
     volumes:
-      - ".:/plugin:ro"
```

Comment:
Do we need to mount `/plugins` read-write?

Reply:
@scadu Not really, it can simplify tests as did get hit with a Permissions Denied issue when looking to re-do all the tests at once 
But side note we've got a mixture of using both `read-write` and `read-only` with mounts for testing across other Plugins but can revert this.

Reply:
@tomowatt Just curious if that was required for the plugin to make it work as it's `ro` now. Wouldn't extend permissions with no clear reason.